include/Nazara/Utility/VertexDeclaration.inl

```diff
@@ -15,6 +15,17 @@ namespace Nz
 
 		return object.release();
 	}
+
+	template<typename T> 
+	bool VertexDeclaration::HaveComponent(VertexComponent component) const
```

Comment:
HasComponent*

Reply:
Plus, I think we should have two methods here, "HasComponent" and "HasComponentOfType<T>".